Makefile

```diff
@@ -1,7 +1,7 @@
 # Current Operator version
-VERSION ?= 0.5.0
```

Comment:
would it make sense to follow the same pattern as for the operator, to use the git-tag?

Reply:
I reset it to `0.0.1` and will be overridden at runtime